### ISSUE TITLE
[BUG] Validation error in lightgbm model

### ISSUE THREAD

**Issue by sharabhshukla** [unit8co/darts#703]
**Describe the bug**
<img width="1137" alt="Screenshot 2021-12-22 at 4 02 57 PM" src="https://user-images.githubusercontent.com/60461107/147079108-ba685caa-7463-4a0f-90ab-4b1f5add4b56.png">

**To Reproduce**
Steps to reproduce the behavior, preferably code snippet.

**Expected behavior**

**System (please complete the following information):**
 - Python version:  3.7.12
 - darts version: 0.14.0

**Additional context**

The .fit attribute works fine on other system, I am running one on my M1 Mac, it only throws up error when running on colab
On colab, the error goes away when not passing validation series, but validation series, it quite important to prevent overfitting.

**Comment by bawiek** [comment]
Hi, similar problem here, but this is caused by validation set.

Validation only works with output_chunk_length=1, otherwise I got the same error.

Darts version: 0.20.0

Simple code example to Reproduce :
`from darts.datasets import AirPassengersDataset
series_air = AirPassengersDataset().load()
scaler_air, scaler_milk = Scaler(), Scaler()
series_air_scaled = scaler_air.fit_transform(series_air)
train_air, val_air = series_air_scaled[:-36], series_air_scaled[-36:]

model_lightGBM = LightGBMModel( lags=10, 
                                lags_past_covariates=None, 
                                lags_future_covariates=None, 
                                output_chunk_length=2, 
                                likelihood=None, 
                                quantiles=None, 
                                random_state=None)
model_lightGBM.fit(
    series = train_air,
    val_series = val_air,
    #past_covariates = None,
    #val_past_covariates = None,
    #future_covariates = None,
    #val_future_covariates = None,
    verbose=True
)`

**Comment by hrzn** [comment]
Will be fixed with https://github.com/unit8co/darts/pull/1154
